### ISSUE TITLE
TensorBoard showing nothing after upgrade 

### ISSUE THREAD

**Issue by yomcoding** [tensorflow/tensorflow#36083]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): yes
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): windows 1809
- TensorFlow installed from (source or binary): binary
- TensorFlow version (use command below): v2.1.0-rc2-17-ge5bf8de410
- Python version: 3.7.3
- CUDA/cuDNN version: cuda_10.1.243_426.00_win10 / cudnn-10.1-windows10-x64-v7.6.5.32
- GPU model and memory: gtx 1070 / 6g

**Describe the current behavior**
I log some data by tf.summary.scalar, every things is ok UNTILL i upgrade tf from 2.0.0 to 2.1.0
through pip install -upgrade tensorflow.
Then I use TensorBoard but both chrome and edge show nothing.
I can get correct result by TensorBoard in tf 2.0.0 which is installed on other mechine.
Then i try to reinstall tensorboard by pip uninstall tensorboard then pip install tensorboard,
but still not working.

**Describe the expected behavior**
Web page show nothing on both chrome and edge.

**Code to reproduce the issue**
I run tensorboard by CMD: tensorboard --logdir=log

**Other info / logs**
CMD show
2020-01-21 12:26:21.211807: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_101.dll
Serving TensorBoard on localhost; to expose to the network, use a proxy or pass --bind_all
TensorBoard 2.1.0 at http://localhost:6006/ (Press CTRL+C to quit)

**Comment by ravikyram** [comment]
@yomcoding 

Request you to provide the simple standalone code to reproduce the issue in our environment. It helps us in localizing the issue faster. Thanks!

**Comment by yomcoding** [comment]
I log the loss value by 
```
file_writer = tf.summary.create_file_writer(self.logs_path)
file_writer.set_as_default()
# ......
loss_value = loss(y_train[step], y_pred)
tf.summary.scalar("Loss",loss_value,step=cur_stp + step)
# ......
```

Then i got log file, but when I open webpage, it shows nothing:
![result](https://fn.storage.img.ztzl.moe/2020/01/22/efa4ed8e1826bc98e2070273c7884a12.jpg)

How can I get log info of tensorboard?

**Comment by gowthamkpr** [comment]
Yes, I tried running on colab notebook, I ran into the error too. Here is my [gist](https://colab.research.google.com/gist/gowthamkpr/85159b9cdabb3190d1825d6991af04e6/tensorboard_in_notebooks.ipynb)

**Comment by manivaradarajan** [comment]
Sorry you're having trouble. 

Can you please run the diagnose_tensorboard.py script (link below) in the same
environment from which you normally run TensorFlow/TensorBoard, and
paste the output here?

https://github.com/tensorflow/tensorboard/blob/master/tensorboard/tools/diagnose_tensorboard.py

**Comment by yomcoding** [comment]
### Diagnostics

<details>
<summary>Diagnostics output</summary>

``````
--- check: autoidentify
INFO: diagnose_tensorboard.py version 5ee40aab64f17918c57bcfbddac7618d5f85956f

--- check: general
INFO: sys.version_info: sys.version_info(major=3, minor=7, micro=3, releaselevel='final', serial=0)
INFO: os.name: nt
INFO: os.uname(): N/A
INFO: sys.getwindowsversion(): sys.getwindowsversion(major=10, minor=0, build=17763, platform=2, service_pack='')

--- check: package_management
INFO: has conda-meta: False
INFO: $VIRTUAL_ENV: None

--- check: installed_packages
INFO: installed: tb-nightly==1.14.0a20190603
INFO: installed: tensorboard==2.1.0
WARNING: conflicting installations: ['tb-nightly', 'tensorboard']
INFO: installed: tensorflow==2.1.0
INFO: installed: tf-estimator-nightly==1.14.0.dev2019060501
INFO: installed: tensorflow-estimator==2.1.0
WARNING: conflicting installations: ['tensorflow-estimator', 'tf-estimator-nightly']

--- check: tensorboard_python_version
INFO: tensorboard.version.VERSION: '2.1.0'

--- check: tensorflow_python_version
2020-01-24 08:37:23.950783: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_101.dll
INFO: tensorflow.__version__: '2.1.0'
INFO: tensorflow.__git_version__: 'v2.1.0-rc2-17-ge5bf8de410'

--- check: tensorboard_binary_path
INFO: which tensorboard: b'C:\\Program Files (x86)\\Microsoft Visual Studio\\Shared\\Python37_64\\Scripts\\tensorboard.exe\r\n'

--- check: addrinfos
socket.has_ipv6 = True
socket.AF_UNSPEC = <AddressFamily.AF_UNSPEC: 0>
socket.SOCK_STREAM = <SocketKind.SOCK_STREAM: 1>
socket.AI_ADDRCONFIG = <AddressInfo.AI_ADDRCONFIG: 1024>
socket.AI_PASSIVE = <AddressInfo.AI_PASSIVE: 1>
Loopback flags: <AddressInfo.AI_ADDRCONFIG: 1024>
Loopback infos: [(<AddressFamily.AF_INET6: 23>, <SocketKind.SOCK_STREAM: 1>, 0, '', ('::1', 0, 0, 0)), (<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 0, '', ('127.0.0.1', 0))]
Wildcard flags: <AddressInfo.AI_PASSIVE: 1>
Wildcard infos: [(<AddressFamily.AF_INET6: 23>, <SocketKind.SOCK_STREAM: 1>, 0, '', ('::', 0, 0, 0)), (<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 0,
'', ('0.0.0.0', 0))]

--- check: readable_fqdn
INFO: socket.getfqdn(): 'YOM-PC.lan'

--- check: stat_tensorboardinfo
INFO: directory: C:\Users\ADMINI~1\AppData\Local\Temp\.tensorboard-info
INFO: os.stat(...): os.stat_result(st_mode=16895, st_ino=20547673299973826, st_dev=1780807057, st_nlink=1, st_uid=0, st_gid=0, st_size=0, st_atime=1579663853, st_mtime=1579663853, st_ctime=1579243708)
INFO: mode: 0o40777

--- check: source_trees_without_genfiles
INFO: tensorboard_roots (1): ['C:\\Program Files (x86)\\Microsoft Visual Studio\\Shared\\Python37_64\\lib\\site-packages']; bad_roots (0): []

--- check: full_pip_freeze
INFO: pip freeze --all:
absl-py==0.8.0
astor==0.8.0
astroid==2.2.5
autopep8==1.4.4
cachetools==4.0.0
certifi==2019.11.28
cffi==1.13.2
chardet==3.0.4
colorama==0.4.1
cupy==6.3.0
cycler==0.10.0
Cython==0.29.13
easydict==1.9
fastrlock==0.4
gast==0.2.2
google-auth==1.10.0
google-auth-oauthlib==0.4.1
google-pasta==0.1.7
grpcio==1.26.0
h5py==2.10.0
hanziconv==0.3.2
idna==2.8
isort==4.3.21
jsonlines==1.2.0
Keras-Applications==1.0.8
Keras-Preprocessing==1.1.0
kiwisolver==1.1.0
lazy-object-proxy==1.4.2
lxml==4.4.1
Markdown==3.1.1
matplotlib==3.1.1
mccabe==0.6.1
msgpack==0.6.2
numpy==1.17.2
oauthlib==3.1.0
opencv-contrib-python==4.1.1.26
opencv-python==4.1.2.30
opt-einsum==3.0.1
Pillow==6.2.1
pip==19.3.1
plotly==4.3.0
protobuf==3.9.1
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycodestyle==2.5.0
pycparser==2.19
pylint==2.3.1
pyparsing==2.4.2
python-dateutil==2.8.0
PyYAML==5.2
requests==2.22.0
requests-oauthlib==1.3.0
retrying==1.3.3
rsa==4.0
scipy==1.4.1
setuptools==41.2.0
six==1.12.0
tb-nightly==1.14.0a20190603
tensorboard==2.1.0
tensorboardX==1.9
tensorflow==2.1.0
tensorflow-estimator==2.1.0
termcolor==1.1.0
tf-estimator-nightly==1.14.0.dev2019060501
torch==1.3.1
torchvision==0.4.2
typed-ast==1.4.0
urllib3==1.25.7
Werkzeug==0.15.6
wheel==0.33.6
wrapt==1.11.2

``````

</details>

### Suggestion: Fix conflicting installations

Conflicting package installations found. Depending on the order of
installations and uninstallations, behavior may be undefined. Please
uninstall ALL versions of TensorFlow and TensorBoard, then reinstall
ONLY the desired version of TensorFlow, which will transitively pull
in the proper version of TensorBoard. (If you use TensorBoard without
TensorFlow, just reinstall the appropriate version of TensorBoard
directly.)

Namely:

        pip uninstall tb-nightly tensorboard tensorflow tensorflow-estimator tf-estimator-nightly
        pip install tensorflow  # or `tensorflow-gpu`, or `tf-nightly`, ...

### Next steps

Please try each suggestion enumerated above to determine whether it
solves your problem. If none of these suggestions works, please copy
ALL of the above output, including the lines containing only
backticks, into your GitHub issue or comment. Be sure to redact any
sensitive information.

**Comment by manivaradarajan** [comment]
Thanks. It looks like you have a polluted Python environment with conflicting tensorboard versions. See your diagnostic output above.

Can you please try with a clean Python virtualenv? I'm going to close this because this is an installation issue and not an underlying tensorboard issue.

Please reopen if you find an issue with tensorboard.

**Comment by ginsanity15** [comment]
@manivaradarajan I have experienced the same problem and I tried to debug it by running the code provided by you. However, it asked me to copy all the output to the comment (as provided below). Could you please tell me what is the problem in my case? Thanks for your help in advance.

> 
### Diagnostics

<details>
<summary>Diagnostics output</summary>

``````
--- check: autoidentify
INFO: diagnose_tensorboard.py version d515ab103e2b1cfcea2b096187741a0eeb8822ef

--- check: general
INFO: sys.version_info: sys.version_info(major=3, minor=7, micro=6, releaselevel='final', serial=0)
INFO: os.name: nt
INFO: os.uname(): N/A
INFO: sys.getwindowsversion(): sys.getwindowsversion(major=10, minor=0, build=17134, platform=2, service_pack='')

--- check: package_management
INFO: has conda-meta: True
INFO: $VIRTUAL_ENV: None

--- check: installed_packages
INFO: installed: tensorboard==2.1.0
INFO: installed: tensorflow-gpu==2.1.0
WARNING: no installation among: ['tensorflow-estimator', 'tensorflow-estimator-2.0-preview', 'tf-estimator-nightly']

--- check: tensorboard_python_version
INFO: tensorboard.version.VERSION: '2.1.0'

--- check: tensorflow_python_version
2020-04-06 12:44:52.236991: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_101.dll
INFO: tensorflow.__version__: '2.1.0'
INFO: tensorflow.__git_version__: 'v2.1.0-rc2-17-ge5bf8de410'

--- check: tensorboard_binary_path
INFO: which tensorboard: b'C:\\Users\\bobgao\\AppData\\Local\\Continuum\\anaconda3\\envs\\venv\\Scripts\\tensorboard.exe\r\nC:\\Users\\bobgao\\AppData\\Local\\Programs\\Python\\Python36\\Scripts\\tensorboard.exe\r\n'

--- check: addrinfos
socket.has_ipv6 = True
socket.AF_UNSPEC = <AddressFamily.AF_UNSPEC: 0>
socket.SOCK_STREAM = <SocketKind.SOCK_STREAM: 1>
socket.AI_ADDRCONFIG = <AddressInfo.AI_ADDRCONFIG: 1024>
socket.AI_PASSIVE = <AddressInfo.AI_PASSIVE: 1>
Loopback flags: <AddressInfo.AI_ADDRCONFIG: 1024>
Loopback infos: [(<AddressFamily.AF_INET6: 23>, <SocketKind.SOCK_STREAM: 1>, 0, '', ('::1', 0, 0, 0)), (<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 0, '', ('127.0.0.1', 0))]
Wildcard flags: <AddressInfo.AI_PASSIVE: 1>
Wildcard infos: [(<AddressFamily.AF_INET6: 23>, <SocketKind.SOCK_STREAM: 1>, 0, '', ('::', 0, 0, 0)), (<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 0, '', ('0.0.0.0', 0))]

--- check: readable_fqdn
INFO: socket.getfqdn(): 'paladium.UGCT.UGENT.BE'

--- check: stat_tensorboardinfo
INFO: directory: C:\Users\bobgao\AppData\Local\Temp\.tensorboard-info
INFO: os.stat(...): os.stat_result(st_mode=16895, st_ino=7318349396135381, st_dev=3774082503, st_nlink=1, st_uid=0, st_gid=0, st_size=4096, st_atime=1586169846, st_mtime=1586169846, st_ctime=1585913527)
INFO: mode: 0o40777

--- check: source_trees_without_genfiles
INFO: tensorboard_roots (1): ['C:\\Users\\bobgao\\AppData\\Local\\Continuum\\anaconda3\\envs\\venv\\lib\\site-packages']; bad_roots (0): []

--- check: full_pip_freeze
INFO: pip freeze --all:
absl-py==0.9.0
alabaster==0.7.12
argh==0.26.2
asn1crypto==1.3.0
astor==0.8.1
astroid==2.3.3
atomicwrites==1.3.0
attrs==19.3.0
autopep8==1.4.4
Babel==2.8.0
backcall==0.1.0
bcrypt==3.1.7
bleach==3.1.0
cachetools==4.0.0
certifi==2019.11.28
cffi==1.14.0
chardet==3.0.4
cloudpickle==1.2.2
colorama==0.4.3
cryptography==2.8
cycler==0.10.0
decorator==4.4.1
defusedxml==0.6.0
diff-match-patch==20181111
dill==0.3.1.1
docutils==0.16
entrypoints==0.3
flake8==3.7.9
future==0.18.2
gast==0.2.2
google-auth==1.11.2
google-auth-oauthlib==0.4.1
google-pasta==0.1.8
googleapis-common-protos==1.51.0
grpcio==1.27.2
gym==0.16.0
h5py==2.10.0
idna==2.8
imageio==2.8.0
imagesize==1.2.0
importlib-metadata==1.5.0
intervaltree==3.0.2
ipykernel==5.1.4
ipython==7.12.0
ipython-genutils==0.2.0
ipywidgets==7.5.1
isort==4.3.21
jedi==0.15.2
Jinja2==2.11.1
jsonschema==3.2.0
jupyter==1.0.0
jupyter-client==5.3.4
jupyter-console==6.1.0
jupyter-core==4.6.1
Keras-Applications==1.0.8
Keras-Preprocessing==1.1.0
keyring==21.1.0
kiwisolver==1.1.0
lazy-object-proxy==1.4.3
libtiff==0.4.2
Markdown==3.2.1
MarkupSafe==1.1.1
matplotlib==3.1.3
mccabe==0.6.1
mistune==0.8.4
nbconvert==5.6.1
nbformat==5.0.4
networkx==2.4
notebook==6.0.3
numpy==1.18.1
numpydoc==0.9.2
oauthlib==3.1.0
odl==0.7.0
opt-einsum==3.1.0
packaging==20.1
pandocfilters==1.4.2
paramiko==2.6.0
parso==0.5.2
pathtools==0.1.2
pexpect==4.8.0
pickleshare==0.7.5
Pillow==7.0.0
pip==20.0.2
pluggy==0.13.1
prometheus-client==0.7.1
promise==2.3
prompt-toolkit==3.0.3
protobuf==3.11.3
psutil==5.6.7
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycodestyle==2.5.0
pycparser==2.19
pydocstyle==4.0.1
pydot-ng==2.0.0
pyflakes==2.1.1
pyglet==1.5.0
Pygments==2.5.2
pylint==2.4.4
PyNaCl==1.3.0
pyOpenSSL==19.1.0
pyparsing==2.4.6
pyrsistent==0.15.7
PySocks==1.7.1
python-dateutil==2.8.1
python-jsonrpc-server==0.3.4
python-language-server==0.31.9
pytz==2019.3
PyWavelets==1.1.1
pywin32==227
pywin32-ctypes==0.2.0
pywinpty==0.5.7
PyYAML==5.3
pyzmq==18.1.1
QDarkStyle==2.8
QtAwesome==0.6.1
qtconsole==4.6.0
QtPy==1.9.0
requests==2.22.0
requests-oauthlib==1.3.0
rope==0.16.0
rsa==4.0
Rtree==0.9.3
scikit-image==0.16.2
scipy==1.4.1
Send2Trash==1.5.0
setuptools==45.2.0.post20200210
six==1.14.0
snowballstemmer==2.0.0
sortedcontainers==2.1.0
Sphinx==2.4.0
sphinxcontrib-applehelp==1.0.1
sphinxcontrib-devhelp==1.0.1
sphinxcontrib-htmlhelp==1.0.2
sphinxcontrib-jsmath==1.0.1
sphinxcontrib-qthelp==1.0.2
sphinxcontrib-serializinghtml==1.1.3
spyder==4.1.1
spyder-kernels==1.9.0
tensorboard==2.1.0
tensorflow-datasets==2.0.0
tensorflow-examples===b9e79fa20b42e04efbf2b76e3dd984fc195fab8d-
tensorflow-gpu==2.1.0
tensorflow-gpu-estimator==2.1.0
tensorflow-metadata==0.21.1
termcolor==1.1.0
terminado==0.8.3
testpath==0.4.4
tornado==6.0.3
tqdm==4.42.1
traitlets==4.3.3
ujson==1.35
urllib3==1.25.8
watchdog==0.10.2
wcwidth==0.1.8
webencodings==0.5.1
Werkzeug==1.0.0
wheel==0.34.2
widgetsnbextension==3.5.1
win-inet-pton==1.1.0
wincertstore==0.2
wrapt==1.12.0
yapf==0.28.0
zipp==2.2.0

``````

</details>

### Next steps

No action items identified. Please copy ALL of the above output,
including the lines containing only backticks, into your GitHub issue
or comment. Be sure to redact any sensitive information.

**Comment by sataliulan** [comment]
hi, bro. I've got the same issue today, my original tensorboard verion is 2.1.0, but my tensorflow version is 1.15. I downgraded the tensorboard verion as 2.0.0, got an error like " ERROR: tensorflow-gpu 1.15.0 has requirement tensorboard<1.16.0,>=1.15.0, but you'll have tensorboard 2.0.0 which is incompatible." ,so I install a new version tensorboard  fowllowing the cmd: pip install tensorboard==1.15, then, problem solved! Hope helpful for you.
